Final2-Another Candy Crush Clone/edu/kit/informatik/matchthree/tests/MoveFlipDownTest.java

```diff
@@ -73,12 +73,14 @@ public void applyAndReverse() {
     public void getAffectedPositions() {
 
         Board board = new MatchThreeBoard(Token.set("AB"), 3, 3);
-        Move flipDown = new MoveFactoryImplementation().flipDown(Position.at(0, 2));
-        
+        // changed initial position, as the move would have been illegal
+        // otherwise (or was this intended?)
```

Comment:
personally, I understand this sentence
> Gibt alle Positionen auf dem Spielbrett board zurück, die von dem Zug betroffen sein werden, also verändert werden.
in the way that an empty set should be returned by an illegal move, as it doesn't change the board upon application.